### PR TITLE
newsletter for 1.6.0 articles

### PR DESCRIPTION
**Pull request by infeo**

### CODE REVIEW

File: src/content/desktop-1.6.0-release.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Hello Community,
+We are happy to announce that the Cryptomator Update 1.6.0 will be released very soon.
```
Comment (overheadhunter) [inline src/content/desktop-1.6.0-release.md]: If this is an intended line break, be aware that the rendered markdown will result in a single paragraph.

Comment (infeo) [reply src/content/desktop-1.6.0-release.md]: oh, right. Will correct that, thanks for looking at it!

File: src/content/desktop-1.6.0-release.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+Hello Community,
+We are happy to announce that the Cryptomator Update 1.6.0 is released today! 🎉
```
Comment (overheadhunter) [inline src/content/desktop-1.6.0-release.md:2]: If this is an intended line break, be aware that the rendered markdown will result in a single paragraph.

**Comment by tobihagemann** [comment]
Thanks for the PR! I'll merge this and make corrections in commits independent from this PR. Since the release is very soon, I just want to make quick changes. :smile:

**Pull request merged**